scannercli.properties

```
@@ -17,6 +17,15 @@ flavors.any.label=Any (Requires a pre-installed JVM)
 flavors.docker.label=Docker
 
 
+4.8.description=Update embedded JRE 11 to the latest, bug fixes
+4.8.date=2022-02-06
+4.8.changelogUrl=https://sonarsource.atlassian.net/issues/?jql=project%20%3D%2010143%20AND%20fixVersion%20%3D%2012892%20ORDER%20BY%20priority%20DESC%2C%20key%20ASC
```

Removing the useless filters:
```suggestion
4.8.changelogUrl=https://sonarsource.atlassian.net/issues/?jql=project%20%3D%2010143%20AND%20fixVersion%20%3D%2012892
```